tests/test_knowledge_panels.py

```diff
@@ -128,6 +129,22 @@ def test_data_quality_kp_with_country(monkeypatch):
     result = app.main.data_quality_kp(
         facet="country", value="Turkey", country="Hungary"
     )
+    first_element = result["Quality"]["elements"][0]
+    expected_text = """
+    <p>The total number of issues are 125,here couples of issues</p>
+    <ul>
+        <li>
+            <a href="https://tr-en.openfoodfacts.org/data-quality/ecoscore-production-system-no-label">1396 products with ecoscore-production-system-no-label</a>
+        </li>
+        <li>
+            <a href="https://tr-en.openfoodfacts.org/data-quality/no-packaging-data">1348 products with no-packaging-data</a>
+        </li>
+        <li>
+            <a href="https://tr-en.openfoodfacts.org/data-quality/ecoscore-packaging-packaging-data-missing">1331 products with ecoscore-packaging-packaging-data-missing</a>
+        </li>
+    </ul>
+    """
+    first_element["text_element"] = tidy_html(expected_text)
```

Comment:
If you do this, your test will certainly pass in any situation, for you are crushing the result with what you except !
You just want to tidy it !

```suggestion
    # tidy html before comparison
    first_element["text_element"] = tidy_html(first_element["text_element"])
```